src/ext/conversion/export-properties.ts

```diff
@@ -44,6 +44,8 @@ const exportProperties = (propertyTree: PropTree, hyperCubePath?: string): Expor
   setValue(expFormat, 'properties.qHyperCubeDef', qHyperCubeDef);
 
   conversion.quarantineProperty(expFormat.properties, 'qHyperCubeDef.columnWidths', 'straightTableColumnWidths');
+  conversion.quarantineProperty(expFormat.properties, 'qHyperCubeDef.qColumnOrder', 'straightTableColumnOrder');
```

Comment:
qHyperCubeDef.qColumnOrder and qHyperCubeDef.columnOrder are same?

Reply:
yes, I assume columnOrder is an old property that has been replaced by qColumnOrder. We still need to support columnOrder for the old object to work, I think.

Reply:
you are right, sn-table->old-table->sn-table